test/integration/replicaset/replicaset_test.go

```diff
@@ -348,7 +326,7 @@ func testPodControllerRefPatch(t *testing.T, c clientset.Interface, pod *v1.Pod,
 	}
 }
 
-func markPodsRunningAndReady(t *testing.T, clientSet clientset.Interface, pods *v1.PodList, ns string) {
+func setPodsStatus(t *testing.T, clientSet clientset.Interface, pods *v1.PodList, conditionStatus v1.ConditionStatus, lastTransitionTime time.Time) {
```

Comment:
This helper function targets a list of pods rather than a single pod. I think its use case can be wider by targeting multiple pods.